wlm_server/config/models.py

```diff
@@ -12,7 +13,8 @@ class Config(models.Model):
 
     class Meta:
         constraints = [
-            models.UniqueConstraint(
-                fields=['id'], name='unique_config'
-            )
+            models.CheckConstraint(
+                check=~Q(id__gt=1),
```

Comment:
This might be dangerous, because if we delete a record, then the newly created one will have ID 2. I tested it and I cannot add a record after deleting the previous one.
Maybe we can use [`clean()`](https://docs.djangoproject.com/en/5.1/ref/models/instances/#django.db.models.Model.clean).
See also: https://stackoverflow.com/questions/2138408/limit-number-of-model-instances-to-be-created-django/6436008

Reply:
Wow, what an interesting problem😅 As you recommended, I will use `clean()`.